frontend/src/pages/home/projects/CreateProjectCard.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  Bullseye,
+  Button,
+  CardBody,
+  Flex,
+  FlexItem,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { ProjectObjectType, SectionType, typedObjectImage } from '~/concepts/design/utils';
+import TypeBorderedCard from '~/concepts/design/TypeBorderedCard';
+
+interface CreateProjectCardProps {
+  allowCreate: boolean;
+  onCreateProject: () => void;
+}
+
+const CreateProjectCard: React.FC<CreateProjectCardProps> = ({ allowCreate, onCreateProject }) => (
+  <TypeBorderedCard
+    data-testid={allowCreate ? 'create-project-card' : 'request-project-card'}
+    sectionType={SectionType.organize}
+  >
+    <CardBody>
+      <Bullseye>
+        <Flex direction={{ default: 'column' }} alignItems={{ default: 'alignItemsCenter' }}>
+          <FlexItem>
+            <img
+              src={typedObjectImage(ProjectObjectType.project)}
+              alt="Add project"
+              width={54}
+              height={54}
+            />
+          </FlexItem>
+          {allowCreate ? (
+            <FlexItem>
+              <Button variant="link" isInline onClick={onCreateProject}>
+                Create project
+              </Button>
+            </FlexItem>
+          ) : (
+            <>
+              <FlexItem>
+                <TextContent>
+                  <Text component="h3">Need another project?</Text>
```

Comment:
This font size looks really large relative to other contents on the page.
I'm also not seeing an `h2` level heading after the `h1` for the "Projects" section header. 

<img width="1210" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/21063328/9d039dbf-f48e-493c-afe6-4f35402c80d3">


One suggestion here is to use the `h2` combined with a utility class for the font size:
```
<Text component="h3" class="pf-v5-u-font-size-sm" >Need another project?</Text>
```
When I look at the current design document, the `sm` size is used for this text.